src/components/compounds/AddComment/index.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+import PropTypes from 'prop-types';
+
+import { addComment } from 'Actions/commentActions';
+
+import InputField from 'Atoms/InputField/InputField';
+import { showModal } from 'Actions/authUser';
+import Alert from 'Utils/helpers/alert';
+import './style.scss';
+
+export class AddComment extends React.Component {
+  constructor(props) {
```

Comment:
I don't think there's any need for using `constructor` here since we're already making use of `@babel/plugin-proposal-class-properties` on the application.
You can call the set the initial state like ```
state = {
    content: ''
}; ```

Reply:
Alright, Thanks.
I will sure see to that. 👍